csharp/Platform.Data.Doublets.Benchmarks/MemoryBenchmarks.cs

```diff
@@ -28,13 +30,17 @@ public static void Setup()
             var memory = new HeapResizableDirectMemory();
             _unitedMemory = new UnitedMemoryLinks<uint>(memory);
             _unitedMemoryLinks = _unitedMemory.DecorateWithAutomaticUniquenessAndUsagesResolution();
+
+            _ffiUnitedMemory = new FFI.UnitedMemoryLinks<uint>("db.links");
+            _ffiUnitedMemoryLinks = _ffiUnitedMemory.DecorateWithAutomaticUniquenessAndUsagesResolution();
         }
 
         [GlobalCleanup]
         public static void Cleanup()
         {
             _splitMemory.Dispose();
             _unitedMemory.Dispose();
+            _ffiUnitedMemory.Dispose();
```

Comment:
Wipe up after yourself. This isn't RAII.
```suggestion
            _ffiUnitedMemory.Dispose();
            File.Delete("db.links");
```